index.html

```diff
@@ -1355,12 +1359,18 @@ <h2 id="state-transitions" class="informative">State transitions</h2>
     group discussion.
   </p>
   <section>
-    <h2>Exposing user information</h2>
+    <h3>Exposing user information</h3>
     <p>
       The <a>user agent</a> should never share information about the user to the web page
       (such as the shipping address) without user consent.
     </p>
   </section>
+  <section>
+    <h3>Incognito Mode</h3>
+    <ul>
```

Comment:
Seems odd to have this as an `ul`. Maybe switch back to paragraph. Also, I think the TAG was trying to nail down what to call "incognito mode" (that's quite Chrome specific terminology, I think).